ui/src/App.tsx

```diff
@@ -1,38 +1,39 @@
 import * as React from 'react';
-import styled from "styled-components";
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link
+} from "react-router-dom";
 
-interface Props {
-   name:
-    string
-}
-
-const Title = styled.h1`
-  font-size: 3em;
-  text-align: center;
-  color: palevioletred;
-`;
+import { Home, ShoeMap } from './pages';
 
-const Button = styled.button`
-  display: flex;    
-  color: palevioletred;
-  font-size: 1em;
-  margin: 1em;
-  padding: 0.25em 1em;
-  border: 2px solid palevioletred;
-  border-radius: 3px;
-  
-`;
+function App() {
+  return (
+    <Router>
+      <div>
```

Comment:
Damn does router expect a single child? lol

We can use `React.Fragment` (https://reactjs.org/docs/fragments.html) here instead of the div for now.

Reply:
Oops didn't see that! nope it shouldn't require it. I swear I copied it straight from the docs though 🤔